docs/demo.md

```diff
@@ -0,0 +1,85 @@
+# Kai Demo
+
+Konveyor AI (kai) is Konveyor's approach to easing modernization of application source code to a new target by leveraging LLMs with guidance from static code analysis augmented with data in Konveyor that helps to learn how an Organization solved a similar problem in the past (Source: https://github.com/konveyor-ecosystem/kai/blob/main/README.md)
+
+In this demo, we are going to migrate a [partially migrated JavaEE application](https://github.com/savitharaghunathan/coolstore.git) to Quarkus, and deploy it in OpenShift
```

Comment:
How about we use the version at https://github.com/konveyor-ecosystem/coolstore instead?

We can push the 'mig' branch there as well.

Reply:
once https://github.com/konveyor-ecosystem/coolstore/pull/30 merges, I will update the PR accordingly. Thanks for the review